src/dymaptic.GeoBlazor.Core/Components/Widgets/Domain.cs

```diff
@@ -77,19 +76,29 @@ public override async Task UnregisterChildComponent(MapComponent child)
 ///     The coded value in a domain.
 ///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-support-CodedValueDomain.html#CodedValue">ArcGIS JS API</a>
 /// </summary>
-public class CodedValue : MapComponent
+public class CodedValue<T> : MapComponent
 {
     /// <summary>
     ///     The name of the coded value.
     /// </summary>
     [Parameter]
     public string? Name { get; set; }
-    
+
     /// <summary>
     ///     The value of the code.
     /// </summary>
     [Parameter]
-    public string? Code { get; set; }
+    public T? Code { get; set; }
+}
+
+/// <summary>
+/// Inherited domains apply to domains at the feature type level. It implies that the domain for a given field at the feature
```

Comment:
I realize this is very nitpicky, but can we use the indented XML style I have above for all XML comments?